### DISCUSSION TITLE
Sample weighting feature?

### DISCUSSION THREAD

**Discussion by kevin-ledezma** [Samuel-Osian-Andrews/GroupThink#3] [Ideas]
Hi Samuel. I think it would be great to have a sample weighting function, like the svydesign function from the “survey” package

**Comment by Sam-Osian** [comment]
Hi Kevin - yes, I absolutely agree! 

Developing an original weighting function is definitely within my plans, but is likely to require a bit of time. In the meantime, I plan on making `unify()` compatible with an existing 'weight' column, that one could generate through another package. This will act as a placeholder of sorts until I'm able to develop a weighting function of my own. 

Best wishes,
Sam

**Comment by kevin-ledezma** [comment]
Hi Sam,
Thanks for the feedback. Could I help you with this? I’m mostly a hobbyist
but it would be really cool.

Thanks,
Kevin

**Comment by Sam-Osian** [comment]
Thank you for your kind offer, Kevin! I'd be very happy to have your support. I'll reach out via email when I have some time so that we can discuss this a bit more!

**Comment by kevin-ledezma** [comment]
Please do! Thanks :)
